src/DotPrint.cc

```diff
@@ -93,12 +93,14 @@ int main(int argc, char *argv[])
     Cairo::RefPtr<Cairo::PdfSurface> cs = Cairo::PdfSurface::create(cmdline.GetOutputFile(), p.m_Width, p.m_Height);
     assert(cs);
 
-    Margins m(10.0 * milimeter,10.0 * milimeter,10.0 * milimeter);
+    Margins m = cmdline.GetPageMargins();
 
     CairoTTY ctty(cs, p, m, preproc);
 
-    // copy the file into ctty
-    ctty.SetFont(cmdline.GetFontFace(), cmdline.GetFontSize(), Cairo::FONT_SLANT_NORMAL, Cairo::FONT_WEIGHT_NORMAL);
+    // Set the font
+    ctty.SetFontName(cmdline.GetFontFace());
+    ctty.SetFontSize(cmdline.GetFontSize());
+    ctty.UseCurrentFont();
```

Comment:
I was also thinking the same when I was debating whether to use it or not, for the very exact reason you point out: forgetting to call either `UseCurrentFont()` or one of the configuration setters.

I decided to go on with it because of the way `CairoTTY` sets a font from what I initially understood: `m_Context->select_font_face(family, slant, weigth);` seems to me that it is loading the exact font that has both the desired `slant` and `weight`, but I do not see an obvious way of changing both parameters "on the fly"---save having to load a new font again via executing `m_Context->select_font_face` again.

As for not including `UseCurrentFont()` at the end of each configuration setter, I thought it would be a waste to set an entire new font every time a configuration parameter changed, just to have it lost as other parameter was dealt with. That is why I opted for delegating to the user the responsibility of actually setting the font after fiddling with the available configuration parameters.

Nonetheless, I welcome any suggestions on how to deal with this in a more efficient manner. :+1: